doc/mesh-adapter-api.txt

```diff
@@ -0,0 +1,108 @@
+BlueZ D-Bus Mesh Adapter API description
+********************************
+
+Overview
+========
+
+This document describes API for mesh adapters. It allows granting and
+revoking access for MicroGateway and reading connection statistics.
+
+
+AccessControlList Hierarchy
+===========================
+Service		org.bluez.mesh
+Interface	org.bluez.mesh.AccessControlList1
+Object path	/org/bluez/mesh/{tcpserver_<port>}
+                                where <port> is a TCP port on which
+                                TCP server is listening.
+
+Methods:
+    uint64 token GrantAccess(array{byte}[16] uuid,
+                             array{byte}[16] dev_key,
+                             array{byte}[16] net_key)
+
+        Grants access to an internal TCP server to a microGateway defined by
+        its UUID & device and network keys. Computes an identity and
+        a pre-shared key based on these arguments.
+        a pre-shared-key. Computed values with generated random token are
+        permanently stored in the configuration file as an ACL entry.
+
+        The token parameter is a 64-bit number that has been assigned to
+        the access entry when it was created. The daemon uses the token
+        to verify whether the application is authorized revoke access.
+
+        array{byte}[16] uuid
+
+            UUID of the MicroGateway.
+
+        array{byte}[16] dev_key
+
+            Device Key of the MicroGateway.
+
+        array{byte}[16] net_key
+
+            Network Key of the network on which MicroGateway is provisioned.
+
+        PossibleErrors:
+            org.bluez.mesh.Error.AlreadyExists
+            org.bluez.mesh.Error.InvalidArguments
+            org.bluez.mesh.Error.Failed
+
+    void RevokeAccess(uint64 token)
+
+        Removes the ACL entry identified by the 64-bit token parameter.
+        The token parameter has been obtained as a result of successful
+        GrantAccess() method call.
+
+        PossibleErrors:
+            org.bluez.mesh.Error.InvalidArguments
+            org.bluez.mesh.Error.NotFound
+            org.bluez.mesh.Error.Failed
+
+
+Connection State Hierarchy
+==========================
+Service		org.bluez.mesh
+Interface	org.bluez.mesh.ConnectionState1
+Object path	/org/bluez/mesh/<adapter>/<name>
+                where <adapter> is an adapter identifier e.g. {tcpserver_65254}
+                <name> is connection name for TCPServer it is an identity.
```

Comment:
Rename `<name>` placeholder to `<identity>`, reword to: "`<identity>` is client TLS identity hint, calculated after calling `GrantAccess.

Question: since GrantAccess returns just the token, how should one obtain object path for the created connection? I'd like the key derivation to be handled internally, so maybe it 'd better to return token *and* object path from this method?